### ISSUE TITLE
Can this project run vehicle reid on dataset VeRi or other vehicle datasets？

### ISSUE THREAD

**Issue by twj1999** [layumi/Person-reID_GAN#18]
hi, author.Can this project run vehicle reid on dataset VeRi or other vehicle datasets？

作者您好，请问这个项目可以跑车辆再识别吗，在VeRi数据集上或其他数据集

**Comment by layumi** [comment]
Yes, you may refer to my latest repo https://github.com/NVlabs/DG-Net

**Comment by twj1999** [comment]
> Yes, you may refer to my latest repo https://github.com/NVlabs/DG-Net

thanks, i will try to figure it out.
